modules/currency-l0/src/main/scala/org/tessellation/currency/l0/CurrencyL0App.scala

```diff
@@ -103,6 +115,11 @@ abstract class CurrencyL0App(
         .start(storages, services, programs, queues, healthChecks, services.dataApplication)
         .asResource
 
+      _ <- {
+        implicit val logger: SelfAwareStructuredLogger[IO] = Slf4jLogger.getLoggerFromName[IO]("GlobalL0Service.realignToMajority")
```

Comment:
why the logger's name is `GlobalL0Service`? It's created in CurrencyL0App and used in Daemon, so neither one is `GlobalL0Service` 🤔 
if you need to pass it down to some service, I'd either create the logger when creating a service or pass it explicitly

Reply:
That code isn't there anymore.
However, the reason the logger was named the way it was is because it was used by the `GlobalL0Service.realign...` function.